### PR TITLE
Test for bug #121

### PR DESCRIPTION
**Pull request by Gorott**
Fixes https://github.com/wwosimulation/tracker/issues/121

### CODE REVIEW

**Comment by 3061LRTAGSPKJMORMRT** [comment]
Why would priv-sl exist?

**Comment by Gorott** [comment]
> Why would priv-sl exist?

role aliases exist now and the channel names are the args so it would be priv-sl or priv-sect-leader

**Review by thewilloftheshadow — CHANGES_REQUESTED** [review]
While this change theoretically works, would it not be better to edit the srole.js file so that abbreviations in the channel names aren't created in the first place?

**Comment by Gorott** [comment]
> While this change theoretically works, would it not be better to edit the srole.js file so that abbreviations in the channel names aren't created in the first place?

Yea it would ill try to work on it this just was easier lmao

**Comment by Gorott** [comment]
Ok now its in srole

File: commands/game/convert.js [hunk 1]
```diff
@@ -77,4 +77,4 @@ module.exports = {
             db.set(`bite_${message.channel.id}`, guy.nickname)
         }
     }
-}
\ No newline at end of file
+}
```
Comment (thewilloftheshadow) [inline commands/game/convert.js:80]: Best change ever

File: commands/narrator/srole.js [hunk 2]
```diff
@@ -207,11 +207,10 @@ module.exports = {
         shuffle(finalRoleList)
         let sorcChats = []
         for (let k = 0; k < alive.members.size; k++) {
-            let theirRole = finalRoleList[k]
+            let theirRole = finalRoleList[k].name
             let role = getRole(theirRole)
             rolelist.push(theirRole)
             let guy = message.guild.members.cache.find((x) => x.nickname == `${k + 1}`)
-            db.delete(`suicided_${guy.id}`)
```
Comment (thewilloftheshadow) [inline commands/narrator/srole.js:214]: Again, this line doesn't need to be deleted

File: commands/narrator/srole.js [hunk 3]
```diff
@@ -207,11 +207,10 @@ module.exports = {
         shuffle(finalRoleList)
         let sorcChats = []
         for (let k = 0; k < alive.members.size; k++) {
-            let theirRole = finalRoleList[k]
+            let theirRole = finalRoleList[k].name
```
Comment (thewilloftheshadow) [inline commands/narrator/srole.js:210]: theirRole is always a string iirc, is there a spot that runs getRole later that we can use the name from?

**Review by thewilloftheshadow — CHANGES_REQUESTED** [review]

File: commands/narrator/srole.js (hunk 2 above)
Comment (Gorott) [reply commands/narrator/srole.js:214]: Hold up i deleted that?

**Review by thewilloftheshadow — APPROVED** [review]

**Pull request merged**